bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 14.5.0
+version: 14.5.1
```

Please bump the minor version instead:
```suggestion
version: 14.6.0
```